sympy/solvers/solveset.py

```diff
@@ -854,6 +862,9 @@ def solveset(f, symbol=None):
 
     f = sympify(f)
 
+    if f is S.false:
```

Comment:
Reason is `solve(0, x)`  returns `[ ]` but `solveset(0, x)` returns `(-oo, oo)` . At  the top of `solveset` module it is written that `solveset` returns a `set` not mentioning of boolean.